simplesat/package.py

```diff
@@ -68,6 +71,10 @@ def version(self):
     def dependencies(self):
         return self._dependencies
 
+    def __repr__(self):
+        return "PackageMetadata('{0}-{1}', key={2!r})".format(
+            self._name, self._version, self._key)
```

Comment:
Key in the original code had a completely different meaning (was the egg filename, keeping the casing information lost in the name)